### ISSUE TITLE
Add deployment option via CVP

### ISSUE THREAD

**Issue by carlbuchmann** [aristanetworks/ansible-avd#18]

**Comment by titom73** [comment]
Because `ansible-cvp` requires a specific input structure for `cv_container` `cv_configlet` and `cv_device`, a module is part of the repository to transform data from inventory to correct struture:

```yaml
all:
  children:
    CVP:
      hosts:
        cvp:
# DC1_Fabric - EVPN Fabric running in home lab
    DC1:
      children:
        enterprise_servers:
          hosts:
            srv01:
        DC1_FABRIC:
          children:
            DC1_SPINES:
              hosts:
                DC1-SPINE1:
                DC1-SPINE2:
            DC1_L3LEAFS:
              children:
                DC1_LEAF1:
                  hosts:
                    DC1-LEAF1A:
                DC1_LEAF2:
                  hosts:
                    DC1-LEAF2A:
                    DC1-LEAF2B:
                DC1_SVC3:
                  hosts:
                    DC1-SVC3A:
                    DC1-SVC3B:
                DC1_BL1:
                  hosts:
                    DC1-BL1A:
                    DC1-BL1B:
```

__Modul inputs__

```yaml
- name: generate intented variables
  inventory_to_container:
    inventory: 'inventory.yml'
    container_root: 'DC1_FABRIC'
    destination: 'generated_vars/{{inventory_hostname}}.yml'
  register: CVP_VARS
```

__Output is currently like:__

```yaml
inventory_to_cv_topology:
      CVP_TOPOLOGY:
        DC1_BL1:
          devices:
          - DC1-BL1A
          - DC1-BL1B
          parent_container: DC1_L3LEAFS
        DC1_FABRIC:
          parent_container: Tenant
        DC1_L2LEAF4:
          devices:
          - DC1-L2LEAF4A
          parent_container: DC1_L2LEAFS
        DC1_L2LEAF5:
          devices:
          - DC1-L2LEAF5A
          - DC1-L2LEAF5B
          parent_container: DC1_L2LEAFS
```

**Comment by carlbuchmann** [comment]
resolved by #41
